ban/db/fields.py

```diff
@@ -151,6 +151,21 @@ def coerce(self, value):
         return value
 
 
+class FantoirField(CharField):
+
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 10
```

Comment:
Let's try to override the vaidator max_length through the `resource_schema` Class property.